sys/net/gnrc/link_layer/lorawan/gnrc_lorawan_mcps.c

```diff
@@ -357,6 +374,16 @@ void gnrc_lorawan_event_no_rx(gnrc_lorawan_t *mac)
         return;
     }
 
+    if ((mac->mcps.adr_ack_cnt  > (CONFIG_LORAMAC_DEFAULT_ADR_ACK_LIMIT +
```

Comment:
IIUC the standard tells that after ACK_LIMIT the device keeps sending ADRAckReq bits until it reaches ACK_LIMIT + ADR_ACK_DELAY. Then it keeps stepping down every time ADR_ACK_DELAY is reached. As it is, this snippet steps down every "ADR_ACK_LIMIT" + "ADR_ACK_DELAY".

Also, this wouldn't reset the counter if `last_dr == 0`

Reply:
Yes, you are right. I shall fall back to `CONFIG_LORAMAC_DEFAULT_ADR_ACK_LIMIT` instead.